test/format/format_config_def.c

```diff
@@ -244,6 +250,9 @@ CONFIG configuration_list[] = {{"assert.read_timestamp", "assert read_timestamp"
 
   {"runs.rows", "number of rows", C_TABLE, 10, M(1), M(100), V_TABLE_RUNS_ROWS},
 
+  {"runs.predictable_replay", "configure predictable replay", C_BOOL, 0, 0, 0,
+    V_GLOBAL_RUNS_PREDICTABLE_REPLAY},
```

Comment:
Alphabetize this above. (Although I think this is generated from the other file.)

Reply:
This one was missed too. I thought it was generated from config.sh but maybe not. Please fix the order.

Reply:
Yes, this is generated.  I forgot to run config.sh, which CMake doesn't check or do.

Reply:
Fixed, now.